SpatialGDK/Documentation/index.md

```diff
@@ -1,11 +1,18 @@
-# The SpatialOS Unreal GDK (pre-alpha)
+# Welcome to the SpatialOS GDK for Unreal 
+
+<img src="{{assetRoot}}assets/unrealgdk-headline-image.png" style=" margin: 0 auto; display: block; width: 60%" />
+
+<br/>
 
 <%(TOC)%>
 
-Welcome to the SpatialOS and Unreal GDK community.
+The SpatialOS Game Development Kit (GDK) for Unreal is a <a href="" data-track-link="Plugin Link Clicked|product=Docs|platform=Win" target="_blank">plugin</a> (Unreal documentation) which allows you to host your game and combine multiple dedicated server instances across one seamless game world, whilst using the Unreal Engine networking API. 
 
-Our SpatialOS community shapes the functionality of the Unreal GDK, so make sure you are part of the conversation. Come and find us on [Discord](https://discordapp.com/channels/311273633307951114/339471548647866368) or talk to us on the **Feedback** section in our [forums](https://forums.improbable.io/) (use the **unreal-gdk** tag - [this link](https://forums.improbable.io/new-topic?category=Feedback&tags=unreal-gdk) takes you there and pre-fills the category and tag). You can also [sign up for SpatialOS Unreal GDK updates](http://go.pardot.com/l/169082/2018-06-15/27ld2t) direct to your mailbox.
+The GDK offers:
+* **Multi-server support:** leveraging our cloud platform [SpatialOS](https://docs.improbable.io/reference/latest/shared/concepts/spatialos) (SpatialOS documentation), the GDK turns Unreal’s single-authority engine into a distributed one,  enabling you to use multiple Unreal servers to make games with more players, Actors, and gameplay systems than previously possible.
```

Comment:
That link thing again! (Sorry, it's been evolving over the last couple of weeks and has sort of solidified now.)

"* **Multi-server support:** leveraging our cloud platform [SpatialOS (SpatialOS documentation)](https://docs.improbable.io/reference/latest/shared/concepts/spatialos),..."